docs/src/managing-packages.md

```diff
@@ -118,9 +118,10 @@ For unregistered packages we could have given a branch name (or commit SHA1) to
 ### Adding a local package
 
 Instead of giving a URL of a git repo to `add` we could instead have given a local path to a git repo.
-This works similarly to adding a URL. The local repository will be tracked (at some branch) and updates
+This works similar to adding a URL. The local repository will be tracked (at some branch) and updates
 from that local repo are pulled when packages are updated.
-Note that changes to files in the local package repository will not immediately be reflected when loading that package.
+Note tracking a package through `add` is distinct from `develop`:
```

Comment:
```suggestion
Note that tracking a package through `add` is distinct from `develop`:
```

Reply:
Oh, you removed the `that` again later. Imo it should either have the "that" or be something like "Note:"